test/gui/shared/steps/steps.py

```diff
@@ -536,22 +543,31 @@ def createPublicShare(context, resource, role):
     clickButton(waitForObject(radioObjectName))
     clickButton(waitForObject(names.oCC_ShareLinkWidget_createShareButton_QPushButton))
 
-@When('the user creates a new public link for folder "|any|" using the client-UI with these details:')
-def step(context, resource):
-    role = ''
-    for row in context.table:
-        if row[0] == 'role':
-            role=row[1]
-            break
+@When('the user creates a new public link for folder "|any|" using the client-UI with these details:')    
+def step(context, resource):    
+    role = ''    
+    for row in context.table:    
+        if row[0] == 'role':    
+            role=row[1]    
+            break    
 
-    if role == '':
-        raise Exception("No role has been found")
-    else:
-        createPublicShare(context, resource, role)
+    if role == '':    
+        raise Exception("No role has been found")    
+    else:    
+        createPublicShare(context, resource, role)    
```

Comment:
No changed done here so fix the indentation.

Reply:
Yes, but for some reason, the IDE changes it.